scalafix-tests/shared/src/main/scala/org/scalatest_autofix/Matchers.scala

```diff
@@ -1,6 +1,3 @@
-/*
-ignore = true
```

Comment:
A HOCON-parseable comment was enough to please the teskit comment parser. This file was relying that behavior to define a helper that should not be evaluated as an input. This is error-prone as the user might have just forgotten to add the rule to be evaluated, so this PR skips HOCON-parseable comments that do not contain a `rules` or `rule` attribute.